src/content/docs/agile-handbook/sprint-mechanics/sprint-workflow-and-jira-boards.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Sprint workflow"
+template: basicDoc
+topics:
+  - Docs agile handbook
+---
+
+All of our sprint work is tracked in Jira. The workflow depends on what type of work we're dealing with: Planned or unplanned ("surprise!") work.
+
+## Planned work [#planned_work]
+
+Planned work includes all work that is currently in our backlog or has been added to the current sprint as a result of a Sprint Planning session. This could include writing or updating documentation, research, meeting with SMEs, information architecture, incorporating peer edits, SME review, and so on.
+
+## Unplanned work (surprises!) [#unplanned_work_surprises]
+
+Usually, we get notified of major requests far enough in advance that we can include them in liaison project plans, backlog grooming, and sprint planning. Occasionally, something bigger surprises us that needs emergency support.
+
+Follow this process with new docs asks to assess the scope of work and ensure we address valid docs needs within a reasonable amount of time. Our goal is to treat the sprint as sacred and insulate against "surprise" work that is not absolutely crucial. But we also want to ensure we're providing good internal customer service, and not getting hung up on process niceties for things that are small.
+
+![The best way to deal with interrupts depends on how much time it takes, and whether the interruption is related to work already in the sprint. For things less than 20 minutes, just do it! For larger interrupts, you should get an estimate from the rest of the team.](./images/dealing_with_interrupts.png "Workflow and tips for dealing gracefully with interruptions.")
```

Comment:
This image is unreadable in dark mode.

Reply:
<img width="905" alt="Screen Shot 2021-10-13 at 3 25 48 PM" src="https://user-images.githubusercontent.com/42678939/137220979-e4ef09ea-f551-4888-ba1c-fb26571abe4e.png">